### PR TITLE
Fixed CNF Test Suite project broken link

### PR DESCRIPTION
**Pull request by gauravgahlot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Cloud Native Network Function Working Group (CNF-WG)
 
-The Cloud Native Network Function WG (CNF WG) operates under the aegis of CNCF. The focus of the CNF WG is to define the process around evaluating the cloud nativeness of networking applications, aka CNFs. We collaborate with the [CNF Test Suite project](https://github.com/cncf/cnf-conformance/blob/master/README-testsuite.md) who work on the mechanics of the conformance tests.
+The Cloud Native Network Function WG (CNF WG) operates under the aegis of CNCF. The focus of the CNF WG is to define the process around evaluating the cloud nativeness of networking applications, aka CNFs. We collaborate with the [CNF Test Suite project](https://github.com/cncf/cnf-conformance/blob/main/README.md) who work on the mechanics of the conformance tests.
```
Comment (xmulligan) [inline README.md]:
```suggestion
The Cloud Native Network Function WG (CNF WG) operates under the aegis of CNCF. The focus of the CNF WG is to define the process around evaluating the cloud nativeness of networking applications, aka CNFs. We collaborate with the [CNF Test Suite project](https://github.com/cncf/cnf-testsuite/blob/main/README.md) who work on the mechanics of the conformance tests.
```

**Review by xmulligan — APPROVED** [review]

**Comment by electrocucaracha** [comment]
@gauravgahlot can you also modify `./charter.md` and `./doc/README.md` references?

**Review by jeffsaelens — APPROVED** [review]
LGTM

**Review by taylor — APPROVED** [review]
lgtm.

**Pull request merged**